app/models/vm_scan.rb

```diff
@@ -600,6 +556,32 @@ def abort_retry(*args)
 
   private
 
+  def create_snapshot(vm)
```

Comment:
Since the callers are checking this method's return value, I think it would be better (clearer) for the method to be more explicit with what it returns.